presto-hive/src/test/java/io/prestosql/plugin/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -1288,6 +1271,30 @@ private void testInsertPartitionedBucketedTableWithUnionAll(HiveStorageFormat st
         assertFalse(getQueryRunner().tableExists(getSession(), tableName));
     }
 
+    @Test
+    public void testInsertTwiceToSamePartitionedBucket()
+    {
+        String tableName = "test_duplicated_insert_with_partitioned_bucketed_table";
+        createPartitionedBucketedTable(tableName, HiveStorageFormat.RCBINARY);
+
+        String insertQuery = format("INSERT INTO " + tableName + " " +
+                "SELECT custkey, custkey AS custkey2, comment, orderstatus " +
+                "FROM tpch.tiny.orders " +
+                "WHERE orderstatus = '%s'", "F");
+        assertUpdate(
+                insertQuery,
+                format("SELECT count(*) from orders where orderstatus = '%s'", "F")
+        );
+
+        assertUpdate(
+                insertQuery,
+                format("SELECT count(*) from orders where orderstatus = '%s'", "F")
+        );
+
+        assertUpdate("DROP TABLE " + tableName);
+        assertFalse(getQueryRunner().tableExists(getSession(), tableName));
```

Comment:
this line is redundant

Reply:
It is, although it exists in every test in this file. I will remove it here, as it is redundant, but maybe it would be good to clean it up for all the other tests in a separate pr